src/views/GovernanceTaskList/index.tsx

```diff
@@ -27,36 +32,24 @@ const GovernanceTaskList = () => {
     (state: AppState) => state.systemIntake.systemIntake
   );
 
-  const calculateIntakeStatus = (intake: SystemIntakeForm) => {
-    if (intake.id === '') {
-      return 'START';
-    }
-    if (intake.status === 'DRAFT') {
-      return 'CONTINUE';
+  useEffect(() => {
```

Comment:
This `useEffect` is going to run on every single render. I don't think we want that to happen, so we should pass it an array dependency as the second argument.
```
useEffect(() => {
}, [systemIntake.id])
```

or similar. That will also allow us to remove/change the conditional in the `useEffect`